perceval/github.md

```diff
@@ -16,6 +16,17 @@ In that help banner, we see two different ways of using it:
 * with no credentials
 * with a user token
 
+## Retrieving from GitHub with category param
+
+```bash 
+(perceval) $ perceval github --category issue grimoirelab perceval
+...
+```
+
+You can pass category as parameter which take issue,pull_request,commit and repository when using Github Perceval backend.
```

Comment:
@sarvesh211999 Just a minor comment. `GitHub` backend only accepts either of these (3) `[ issue, pull_request, repository ]` as parameter. 

[REF](https://github.com/chaoss/grimoirelab-perceval/blob/b3031c06bf76f5380db661a7c6c7be590499199c/perceval/backends/core/github.py#L90)

Reply:
I will change the wording.